pages/newsPage.vue

```diff
@@ -0,0 +1,14 @@
+<template>
```

Comment:
I think we should skip this one for MVP

Reply:
I just added it as an empty page to test out the navigation. 
I'll remove it